### ISSUE TITLE
"SyntaxError: Unexpected token ." when using custom gatsby-transformer-contentful-richtext config

### ISSUE THREAD

**Issue by aofolts** [contentful/rich-text#37]
I added the following to `gatsby-config.js`, per the package readme.

However, when I run `gatsby develop` I get an `unexpected token .` error on the require statement.

Is this a valid require syntax? 

```
const { BLOCKS.EMBEDDED_ENTRY } = require ('@contentful/rich-text-types')

plugins: [
  {
   resolve: `gatsby-transformer-contentful-richtext`,
   options: {
      renderOptions: {
        /*
        * Defines custom html string for each node type like heading, embedded entries etc..
        */
        renderNode: {
          // Example
          [BLOCKS.EMBEDDED_ENTRY]: (node) => `<div class='custom-entry'>${customComponentRenderer(node)}</div>`
        },
        /*
        * Defines custom html string for each mark type like bold, italic etc..
        */
        renderMark: {
          // Example
          [MARKS.BOLD]: text => `<custom-bold>${text}<custom-bold>`
        }
      }
   }
  }
]
```

**Comment by aofolts** [comment]
When I change it to `const {BLOCKS}`, and switch to `resolve: @contentful/gatsby-transformer-contentful-richtext` Gatsby compiles alright, but the custom render options don't have any effect.

**Comment by epegase** [comment]
@Khaledgarbaya 
Please, test the gatsby-transformer-contentful-richtext.
For two days, i query the html representation of  rich text field but i see nothing in my localhost/__graphql

**Comment by aofolts** [comment]
@epegase The same thing was happening for me, and then the field suddenly showed up. I have no idea what I did to make it work, but you might try deleting your `.cache` folder—that's my go-to fix-all for Gatsby issues.

**Comment by Khaledgarbaya** [comment]
Hey @epegase 

> Please, test the gatsby-transformer-contentful-richtext.

This can be asked differently in the future maybe :) and as @aofolts said this could be related to `.cache` directory.

For custom render option I will investigate that asap

**Comment by Khaledgarbaya** [comment]
Hey @aofolts,
can you try `const { BLOCKS } = require ('@contentful/rich-text-types')` instead, I think that was a mistake in the docs

**Comment by Khaledgarbaya** [comment]
Hey @aofolts,
My apologies there was a silly mistake in the README.
I prepared a sample project that works with custom renderers
here is the code sandbox [link](https://codesandbox.io/s/jljrl7z82w)

Cheers,
Khaled

**Comment by aofolts** [comment]
@Khaledgarbaya No worries! I tried out the sandbox, and it worked for me. 

I then copied and pasted everything over to my local project, but the custom renderers still don't work. My `package.json` and `gatsby-config.js` are exactly the same as the sandbox example, so it's unclear what the issue could be. 

I'll have to do some more investigating. Thanks for clarifying about the readme, though. I was thinking...is this some new require syntax or what, haha?
